2024/index.html

```diff
@@ -69,7 +69,7 @@ <h1 class="sprite logo wow animated fadeInDown justify-content-center">
                     <meta itemprop="startDate" content="2019-10-18" />
                     <meta itemprop="image" content="static/style/images/favicon.png" />
                     <div class="hello wow animated fadeInUp" data-wow-delay="1.5s">Hello World!</div>
-                    <code>&lt;script src=&quot;CONVOKE6_0.js&quot;&gt;&lt;/script&gt;</code>
+                    <!-- <code>&lt;script src=&quot;CONVOKE6_0.js&quot;&gt;&lt;/script&gt;</code>                 -->
```

Comment:
```suggestion
                    <code>&lt;script src=&quot;CONVOKE6_0.js&quot;&gt;&lt;/script&gt;</code>
```
Already been fixedin #12